subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy

```diff
@@ -265,6 +267,7 @@ it.exclude group: '*', module: 'badArtifact'
         file("build/libs/util-2.5.jar").exists()
     }
 
+    @Requires(TestPrecondition.FIX_TO_WORK_ON_JAVA9)
```

Comment:
There's actually no difference. We use `FIX_TO_WORK_ON_JAVA9` because it's easier to search for it so we fix the marked test later. There's no clear indication that `JDK8_OR_EARLIER` means "should definitely be fixed for Java 9.

Reply:
Fair enough.